src/benchmark_register.cc

```diff
@@ -355,9 +356,17 @@ Benchmark* Benchmark::MinTime(double t) {
   return this;
 }
 
+Benchmark* Benchmark::MinWarmUpTime(double t) {
+  BM_CHECK(t > 0.0);
```

Comment:
>=, i think. given the default is 0.0, i can imagine a user might want to ensure one benchmark _isn't_ run with warmup.

Reply:
So you are suggesting >=.
If yes, good point done.

Reply:
So you proposing to check it for == 0?
Because this is probably is more interesting to the user than ensuring its positive and therefore set. Agree done

Reply:
figured it out. You meant >=0.0. My bad wasn't paying enough attention. Will fix it immediately